pkg/controllers/managementapi/usercontrollers/usercontroller.go

```diff
@@ -22,12 +20,17 @@ import (
 	"github.com/sirupsen/logrus"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/version"
 )
 
 var (
 	all = "_all_"
 )
 
+// currentClusterControllersVersion is the version of the controllers that are run in the local cluster for a particular downstream cluster.
+const currentClusterControllersVersion = "mgmt.cattle.io/current-cluster-controllers-version"
```

Comment:
Nit: You may want to change the prefix ('mgmt.cattle.io') to something else. I choose that in haste, I don't think it's the right long term name.